src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -71,6 +79,45 @@ public JwtHeader(SigningCredentials signingCredentials)
         }
 
         /// <summary>
+        /// Gets or sets the <see cref="IDictionary{TKey, TValue}"/> used to map
+        /// the JWA identifiers with the corresponding security algorithms.
+        /// </summary>
+        public static IDictionary<string, string> InboundAlgorithmMap { get; } = new Dictionary<string, string>
```

Comment:
Ideally, Inbound/OutboundAlgorithmMap would be instance, however we can't instantiate for every token being processed, as this is clearly a performance issue. There is a way to provide instance, but would require a bit of internal logic.